### PR TITLE
Index user rename in es6 when index_es6 is set

### PR DESCRIPTION
**Pull request by hmstepanek**
Rename the user in the elasticsearch6 cluster when the index_es6
feature flag is enabled.

### CODE REVIEW

**Comment by hmstepanek** [comment]
@klemay here is the fix PR for the renaming user issue.

File: h/services/rename_user.py [hunk 1]
```diff
@@ -91,6 +91,8 @@ def _reindex(ids):
             return
 
         request.tm.commit()
+        if request.feature("index_es6"):
+            indexer = index.BatchIndexer(request.db, request.es6, request)
```
Comment (robertknight) [inline h/services/rename_user.py]: `indexer.index` is only called on the ES1 batch indexer!

File: tests/h/services/rename_user_test.py [hunk 2]
```diff
@@ -123,7 +123,10 @@ def test_it_indexes_the_given_ids(self, req, index):
         indexer([1, 2, 3])
 
         batch_indexer = index.BatchIndexer.return_value
-        batch_indexer.index.assert_called_once_with([1, 2, 3])
+        assert 2 == batch_indexer.index.call_count
```
Comment (robertknight) [inline tests/h/services/rename_user_test.py]: Nitpick - Boolean expressions in assert statements are usually written with the actual value on the left and the expected value on the right.

**Comment by hmstepanek** [comment]
Ok I tested locally on admin pages, it looks like it's working.

File: tests/h/services/rename_user_test.py (hunk 2 above)
Comment (hmstepanek) [reply tests/h/services/rename_user_test.py]: fixed.

**Pull request merged**